report/common/utils.py

```diff
@@ -99,16 +99,29 @@ def fmt_pct(val, pts):
 
 def get_list_typo(in_vals, top, sort):
     """
-    read in a list of tups of (type, pct_1, pct_2) and return top X sorted by 'sort', and then 1 - sum(rest) for "Others"
+    read in a list of tups of (type, pct_1, pct_2)
+    and return top X sorted by 'sort', and then 1 - sum(rest) for "Others"
     """
     assert(len(set(len(v) for v in in_vals)) == 1)
 
-    #TODO: revert nan
-    vals = sorted([[(v if i == 0 else 0 if (v is None or is_nan(v)) else v) for i, v in enumerate(t)]
-                   for t in in_vals], key = lambda x : x[sort], reverse=True)
+    # TODO: revert nan
```

Comment:
i re-wrote the logic for the typologies table per HRRP request so need to ignore these, sorry